src/Shells/StrixMusic.Shells.ZuneDesktop/Controls/Views/Collection/ZuneTrackCollection.cs

```diff
@@ -16,154 +20,182 @@ namespace StrixMusic.Shells.ZuneDesktop.Controls.Views.Collection
     /// <summary>
     /// Zune implmenation for the <see cref="ZuneTrackCollection"/>.
     /// </summary>
-    public class ZuneTrackCollection : TrackCollection
+    [INotifyPropertyChanged]
+    public partial class ZuneTrackCollection : CollectionControl<ZuneTrackCollectionItem, ZuneTrackItem>
     {
+        private ObservableCollection<ZuneTrackCollectionItem> _trackItems = new();
+
+        public ZuneTrackCollection()
+        {
+            TrackItems = new ReadOnlyObservableCollection<ZuneTrackCollectionItem>(_trackItems);
+
+            Loaded += ZuneTrackCollection_Loaded;
+            Unloaded += ZuneTrackCollection_Unloaded;
+        }
+
+        private void AttachEvents(ZuneTrackCollectionItem item)
+        {
+            item.PropertyChanged += Item_PropertyChanged;
```

Comment:
If there is only one subscription you shouldn't need methods for attach and detach properties.

Reply:
There's more than one subscription and unsubscription in different places.

Reply:
Ah ok